rts/motoko-rts-tests/src/gc.rs

```diff
@@ -119,8 +119,8 @@ fn test_gc(
         heap.continuation_table_ptr_offset(),
     );
 
-    for _ in 0..3 {
-        gc.run(heap.clone());
+    for _ in 0..0 {
+        gc.run(heap.clone()); /* FIXME: not alignment-preserving! */
```

Comment:
`clone()` needs to be aware of the dynamic heap's 32-byte alignment constraint when doing compacting GC.

Reply:
Seems to work o my Mac, so I reverted this for now.

Reply:
Looks like this was a red herring.